### PR TITLE
Fix pkgconfig libname population in CMakeLists

### PR DESCRIPTION
**Pull request by prashanthr05**
cc @diegoferigo @s-dafarra @paolo-viceconte @GiulioRomualdi

### CODE REVIEW

**Review by diegoferigo — APPROVED** [review]

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -124,6 +120,10 @@ else()
     endif()
 endif()
 
+# Generate a .pc file
+set(INSTALL_PKGCONFIG_DIR "${CMAKE_INSTALL_PREFIX}/share/pkgconfig" CACHE PATH "Installation directory for pkgconfig (.pc) files")
```
Comment (traversaro) [inline src/CMakeLists.txt]: Nitpick as this was not added in this PR but I noticed it know: in general it is not a good idea to make variable that depend on `${CMAKE_INSTALL_PREFIX}` cache variables, as this means that if you modify `CMAKE_INSTALL_PREFIX`, the `INSTALL_PKGCONFIG_DIR` is not automatically updated.
```suggestion
set(INSTALL_PKGCONFIG_DIR "${CMAKE_INSTALL_PREFIX}/share/pkgconfig")
```

Comment (prashanthr05) [reply src/CMakeLists.txt]: @traversaro thanks for noticing!  I can add a commit, if you agree.

Comment (prashanthr05) [reply src/CMakeLists.txt]: Done in https://github.com/ami-iit/openvr/pull/1/commits/76389e92b1dbb9018f0fff06341bd2a5cc786f26

**Review by traversaro — APPROVED** [review]

**Comment by S-Dafarra** [comment]
@traversaro @diegoferigo can you please hit the merge button?

**Comment by traversaro** [comment]
> @traversaro @diegoferigo can you please hit the merge button?
> 

Done

**Comment by S-Dafarra** [comment]
Thanks!

**Pull request merged**